src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/StacksTests.cs

```diff
@@ -33,10 +33,13 @@ public class StacksTests
         private readonly TemporaryDirectory _tempDirectory;
 
         private const string ExpectedModule = @"Microsoft.Diagnostics.Monitoring.UnitTestApp.dll";
+        private static readonly Guid ExpectedModuleVersionId = new Guid("b271d014-2be3-4972-9835-7523a19ecff8");
```

Comment:
We can't hard code the expected guid / method tokens, since they may change with each build.

We should instead use reflection to get them now that we have a project reference.

See this test for an example of getting a `MethodInfo`:
- https://github.com/dotnet/dotnet-monitor/blob/32fa38302fa3293889835062e611ea753b564618/src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/FunctionProbes/FunctionProbesScenario.cs#L96

Reply:
Looks like I need to update the exceptions tests too